src/funfuzz/js/build_options.py

```diff
@@ -43,7 +42,7 @@ def add(self, name, fastDeviceWeight, slowDeviceWeight):  # pylint: disable=inva
     def getRandomSubset(self):  # pylint: disable=invalid-name,missing-docstring,missing-return-doc
         # pylint: disable=missing-return-type-doc
         def getWeight(o):  # pylint: disable=invalid-name,missing-docstring,missing-return-doc,missing-return-type-doc
-            return o['fastDeviceWeight'] if deviceIsFast else o['slowDeviceWeight']
+            return o["slowDeviceWeight"]
```

Comment:
Minor nit but you may want to use single quotes here to remain consistent.

Reply:
We're gradually moving to double quotes throughout the upcoming refactorings. There might be a few moments of overlap.